### DISCUSSION TITLE
Dropdown problem

### DISCUSSION THREAD

**Discussion by codethevoid** [heroui-inc/heroui#650] [Feedback]
Wasn't sure if you guys were aware but the dropdown doesn't work on the main website.

**Comment by tianenpang** [comment]
~~Yeah, same here.~~

~~The `Dropdown` in docs site only appears when `active` and will close when the mouse is released (A11y works fine).~~

~~I built the docs site locally and didn't find any issue and the `Dropdown` work as expected 🤔~~

Hi @rmthomas98 It has been fixed, please try again 🚀

**Reply by BuchananQuantum** [reply]
This issue is still persistent
